website/docs/tutorials/setup-react.md

```diff
@@ -37,82 +29,89 @@ This setup guide is for any project which uses React.
     }
     ```
 
-    :::info
-    If you use any preset, check first if it contains `macros` plugin. These presets already includes `macros` plugin: `react-scripts`
-    :::
+    If you use any preset, check first if it contains `macros` plugin. These presets already includes `macros` plugin: `react-scripts`.
 
-3.  Create `lingui.config.js` file with LinguiJS configuration in root of your project (next to `package.json`). Replace `src` with the directory name where you have source files:
-
-    ```js title="lingui.config.js"
-    /** @type {import('@lingui/conf').LinguiConfig} */
-    module.exports = {
-      locales: ["en", "cs", "fr"],
-      catalogs: [
-        {
-          path: "<rootDir>/src/locales/{locale}/messages",
-          include: ["src"],
-        },
-      ],
-      format: "po",
-    };
-    ```
+:::info
+For those who prefer not to use Babel, Lingui offers the [SWC Plugin](/docs/ref/swc-plugin.md) as an alternative.
```

Comment:
Many projects, such as those using newer versions of NextJS, do not use babel, and so I think we should have a "fork" all the way at the top that leads to separate installation instructions for SWC and for babel. Maybe using Tabs: https://docusaurus.io/docs/markdown-features/tabs)

I have recently set up a nextJS app with the pages router and RSC and I'd love to write docs for that. I'm not sure if that's going to be a separate guide (probably yes). Either way, I think we should warn people as soon as possible that installing babel macros doesn't make sense if you're using SWC.

Reply:
@vonovak thank you for pointing, a separate Next.js article would be the best!

Reply:
I was thinking about RSC support and probably we can document the current PoC `@lingui/react/server` approach until we have a clearer approach.

Reply:
Would be really nice, to have that article. I also would like to warn people in that article  that using global `t` macro would be harmful in the hybrid applications such as NextJs. And thanks to new useLingui macro-hook now we have a much better alternative. I also think if we don't need a global instance of i18n configured, we could simplify a lot initialization in nextjs to something like that: https://github.com/lingui/js-lingui/issues/1339#issuecomment-1385470909